tests/test_entities.py

```diff
@@ -51,7 +51,7 @@ def test_loader_deprecated_required():
         e0 = EntityTest0()
     assert (
         str(ex.value)
-        == "__init__() missing 1 required positional argument: 'fieldThree'"
+        == "EntityTest0.__init__() missing 1 required positional argument: 'fieldThree'"
```

Comment:
:D Why did this change with the version? :D

Reply:
Heh sounds odd, did the __str__ change?